eng/Azure.Management.Test.targets

```diff
@@ -0,0 +1,39 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <RbacSharedSources>$(MSBuildThisFileDirectory)/../sdk/testcommon/Azure.Graph.Rbac/src</RbacSharedSources>
```

Comment:
Is your plan to still move these under "common"? Also we still need to rename at least the Azure.Graph.Rbac directory has it is causes issue because it looks like a track2 library but it isn't.

Reply:
Lets please file a tracking issue to move this after the preview work is out.

Reply:
Use #12209 to track the request.